### PR TITLE
Rip Space MKII (Realized Dimension Shredder)

### PR DESCRIPTION
**Pull request by scionalu**
## About The Pull Request

Moved from https://github.com/vlggms/lobotomy-corp13/pull/1415
Edits are:
- Finished sprites by Mel
- Spell code overhaul
- Extra temp effects
- Extra sounds added

https://github.com/vlggms/lobotomy-corp13/assets/80381079/73eca5f0-6e25-4703-8eb7-6bfe768bb294

Don't mind the shitty camera movement, this was for a meme localtest. It has been smoothened
AND NO, I DIDN'T PUT IN FEDDY, it was just for memes in local

Adds Dimension Ripper (9/5/7/5) from Dimension Shredder (4/0/3/0)
The ability takes hostile mobs in view and does 3 dash attacks to random targets for each mob
1 second prep before attacking for 80 damage each dash
DPS for 1 target is 150, increases with each target since no extra prep, but it will target randomly

## Why It's Good For The Game

https://github.com/vlggms/lobotomy-corp13/assets/80381079/b55ea14f-867b-45f9-96af-fdcfed3c8286

## Changelog
:cl:
add: Added Dimension Shredder Realization
imageadd: realized stuff and their effects
soundadd: attacks for rip space
/:cl:

### CODE REVIEW

**Comment by TaculoTaculo** [comment]
Zased MKII

**Comment by TheBronJameOffical** [comment]
insane

File: code/modules/spells/ability_types/realized.dm [hunk 1]
```diff
@@ -948,3 +948,87 @@
...
+
+/obj/effect/proc_holder/ability/rip_space/Perform(target, mob/living/user)
+	var/list/targets = list()
+	for(var/mob/living/L in view(8, user))
+		if(L.stat == DEAD)
+			continue
+		if(L.status_flags & GODMODE)
+			continue
+		if(user.faction_check_mob(L, FALSE))
+			continue
```
Comment (LanceSmites328) [inline code/modules/spells/ability_types/realized.dm:968]: If this wasn't here >:)
"Oh hey they have dimension shredder"
"OH DEAR GOD"

File: code/modules/spells/ability_types/realized.dm [hunk 2]
```diff
@@ -948,3 +948,87 @@
...
+	new /obj/effect/temp_visual/ripped_space(get_turf(target))
+	playsound(user, 'sound/abnormalities/wayward_passenger/ripspace_hit.ogg', 75, 0)
+	sleep(1)
+	qdel(DE)
+
+/obj/projectile/ripper_dash_effect
+	speed = 0.32
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "ripper_dash"
+	projectile_piercing = ALL
```
Comment (LanceSmites328) [inline code/modules/spells/ability_types/realized.dm:1031]: Why is it a projectile? Im just actually extremely fuckin curious.

Comment (scionalu) [reply code/modules/spells/ability_types/realized.dm:1031]: The diagonal movement just looks better, people didn't like the old one as much
Also made it return nothing on hit

**Comment by scionalu** [comment]
I want to die

**Pull request closed without merging**